cmake/nokia_ngage.cmake

```diff
@@ -0,0 +1,133 @@
+cmake_minimum_required(VERSION 3.16)
+
+enable_language(CXX)
+
+set(GCC_MODULE_COMN_DEFS -D__SYMBIAN32__ -D__GCC32__ -D__EPOC32__ -D__MARM__ -D__MARM_ARMI__)
```

Comment:
These belong in the cmake toolchain file of the ngage-sdk at https://github.com/ngagesdk/ngage-toolchain/blob/main/cmake/ngage-toolchain.cmake

Reply:
I tried and now it fails: `multiple definition of `operator new(unsigned int, void *)`. I look into it!